src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -46,10 +46,49 @@
 
 helps['consumption pricesheet'] = """
     type: group
-    short-summary: Inspect the price sheet by Azure subscription within current or specified billing period.
+    short-summary: Inspect the price sheet by Azure subscription within a billing period.
 """
 
 helps['consumption pricesheet show'] = """
     type: command
-    short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
+    short-summary: Show the price sheet for Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace by Azure subscription within a billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for Azure subscription within a billing period.
+"""
+
+helps['consumption budget'] = """
+    type: group
+    short-summary: Offers ability to create, get, update, and delete budgets for Azure subscription.
+"""
+
+helps['consumption budget list'] = """
+    type: command
+    short-summary: Lists budget for Azure subscription.
+"""
+
+helps['consumption budget show'] = """
+    type: command
+    short-summary: Show budget for Azure subscription.
+"""
+
+helps['consumption budget create'] = """
+    type: command
+    short-summary: Create a budget for Azure subscription.
+"""
+
+helps['consumption budget update'] = """
+    type: command
+    short-summary: Update a budget for Azure subscription.
+"""
+helps['consumption budget delete'] = """
+    type: command
+    short-summary: Delete a budget for Azure subscription.
```

Comment:
I'm not sure that "Azure subscription" is entirely valuable here, but if so, it should be "an Azure subscription" for all these commands.  cc/ @sptramer

Reply:
done